src/main/java/com/wseemann/ecp/core/HttpManager.kt

```diff
@@ -0,0 +1,57 @@
+package com.wseemann.ecp.core
+
+import java.util.concurrent.TimeUnit
+
+import okhttp3.OkHttpClient
+import okhttp3.ConnectionPool
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.ResponseBody
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+
+class HttpManager private constructor() {
```

Comment:
Can you mark this as internal class please? It shouldn't be exposed as an end user API.